packages/fhir-router/src/repo.ts

```diff
@@ -456,6 +456,12 @@ export class MemoryRepository extends FhirRepository<undefined> {
   async search<T extends Resource>(searchRequest: SearchRequest<T>): Promise<Bundle<T>> {
     const { resourceType } = searchRequest;
     const resources = this.resources.get(resourceType) ?? new Map();
+    if (resources.size === 0) {
```

Comment:
Needed to add this since in the case when no resources were in `MemoryRepository` for a given resource type, we would always return an empty searchset `Bundle` even when given an invalid query with unknown search parameters